roles/tackle/defaults/main.yml

```diff
@@ -161,6 +161,24 @@ analyzer_container_limits_memory: "4Gi"
 analyzer_container_requests_cpu: "1"
 analyzer_container_requests_memory: "4Gi"
```

Comment:
I don't think the analyzer needs 4Gi anymore.  @pranavgaikwad can you comment?

Reply:
true, max I have seen it go is 500m so far, 512m in requests should be good.